donut/modules/bodfeedback/templates/bodfeedback.html

```diff
@@ -0,0 +1,65 @@
+{% extends "layout.html" %}
+{% block page %}
+
+<div class="container theme-showcase" role="main">
+
+    <!-- Main jumbotron for a primary marketing message or call to action -->
+    <div class="jumbotron">
+        <img src="{{  url_for('static', filename='images/newdonut.png')  }}" alt="d(O)nut" title="d(O)nut" style="margin: auto; display: block">
```

Comment:
see our current home page for @Thought-Weaver's updated logo image. as a side note, any design comments @Thought-Weaver